ui/src/bootstrap.js

```diff
@@ -30,7 +30,13 @@ const TuringUI = () => (
   </React.StrictMode>
 );
 
-ReactDOM.render(TuringUI(), document.getElementById("root"));
+const container = document.getElementById("root")
+const root = createRoot(container);
+
+root.render(
+    TuringUI()
+);
```

Comment:
React now has a new way of rendering the UI - this change was similarly performed in Merlin https://github.com/caraml-dev/merlin/pull/591/files#diff-72db741eec2a68e8c97269491de7ff9575b2d5a0417293ea79c6966e5b1c7c99L41 and MLP https://github.com/caraml-dev/mlp/pull/100/files#diff-6af1edbc5aa404abe1e77619dabdee53261810f7077ac22200d48d32b00bbd8cL15.